src/scss/utilities/_padding.scss

```diff
@@ -1,11 +1,12 @@
 // e.g ons-u-pl-m
 $sizes: (
-    no: 0 !important,
```

Comment:
!important should be added on line 22 in the css file like in the margin one. Other than that looks good!

Reply:
done